src/FormBuilder.php

```diff
@@ -944,6 +946,25 @@ protected function getModelValueAttribute($name)
 			return array_get($this->model, $this->transformKey($name));
 		}
 	}
+	
+	/**
+	 * Cycle through the Eloquent Collection to see if any
+	 * relationship models include the given input value
+	 *
+	 * @param  \Illuminate\Database\Eloquent\Collection $collection
+	 * @param  string $value
+	 * @param  string $attribute
+	 * @return bool
+	 */
+	protected function checkRelationship($collection, $value, $attribute = 'id')
```

Comment:
If you always pass an instance of `\Illuminate\Database\Eloquent\Collection` to this method, can you please type-hint it in the method signature?